src/MusicStore/Views/ShoppingCart/Index.cshtml

```diff
@@ -6,10 +6,10 @@
 
 @functions
 {
-    public string GetAntiXsrfToken()
+    public string GetAntiXsrfRequestToken()
     {
         var tokens = Xsrf.GetTokens(Context);
```

Comment:
This is getting the XSRF token for use in AJAX below. Are you saying there's a better way to get it on the page or that we can inline it down there?